src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -392,6 +392,15 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             vkey != VK_SNAPSHOT &&
             keyDown)
         {
+            // check for "select all"
+            if (modifiers.IsCtrlPressed() && vkey == 'A')

```

Comment:
Making this configurable would be preferable IMO.

Reply:
Oh man the debates we've had over this 😝